### ISSUE TITLE
401 Unauthorized for transactions

### ISSUE THREAD

**Issue by amnacog** [femueller/python-n26#66]
With:
> n26, version 2.1.0

When I try to get the transactions I get this:
```
$> n26 transactions
Initiating authentication flow, please check your phone to approve login.
Authentication successful :)
Output is limited to 5 entries.
Traceback (most recent call last):
  File "/usr/local/bin/n26", line 10, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.7/dist-packages/Click-7.0-py3.7.egg/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/Click-7.0-py3.7.egg/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/dist-packages/Click-7.0-py3.7.egg/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/dist-packages/Click-7.0-py3.7.egg/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/dist-packages/Click-7.0-py3.7.egg/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/amnacog/.local/lib/python3.7/site-packages/n26/cli.py", line 49, in wrapper
    return func(*args, **kwargs)
  File "/home/amnacog/.local/lib/python3.7/site-packages/n26/cli.py", line 341, in transactions
    categories=categories)
  File "/home/amnacog/.local/lib/python3.7/site-packages/n26/api.py", line 207, in get_transactions
    'lastId': last_id
  File "/home/amnacog/.local/lib/python3.7/site-packages/n26/api.py", line 292, in _do_request
    response.raise_for_status()
  File "/usr/lib/python3/dist-packages/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://api.tech26.de/api/smrt/transactions?limit=5
```

Other commands like status or balance work as expected

**Comment by wasmitnetzen** [comment]
I'm seeing the same error. I tried some variations in the parameters to check if it was the timestamps or the limit, without success so far.

**Comment by femueller** [comment]
This seems to me somehow only related to the transactions API endpoint.

When trying an API endpoint (for example `get_balance()`), everything seems to work just fine with the same request.

**Comment by Sjord** [comment]
The authentication seems to be failing because there is a space missing in the Authorization header. There should be a space between "Bearer" and the token [here](https://github.com/femueller/python-n26/blob/master/n26/api.py#L281).

**Comment by Holden4** [comment]
@Sjord well spotted, added a space and can get them now.
However, it requires authentication through the app everytime the command is run, where as balance will last a full session with out reauthenticating.

**Comment by markusressel** [comment]
@Sjord Thx! Very well spotted indeed :+1: 

@Holden4 I can not confirm that the command requires authentication on every run. In my testing it works after an initial 2FA confirmation without requiring a new one for a second call.

**Comment by markusressel** [comment]
Fix released with 2.1.1

**Comment by Holden4** [comment]
Thanks @markusressel all good now after new release.
